runtime/ftplugin/i3config.vim

```diff
@@ -0,0 +1,12 @@
+" Vim syntax file
+" Language: i3 config file
+" Original Author: Mohamed Boughaba <mohamed dot bgb at gmail dot com>
+" Maintainer: Quentin Hibon
+" Version: 0.4
+" Last Change: 2019-03-23 21:54
+
+if exists("b:did_ftplugin") | finish | endif
+let b:did_ftplugin = 1
+let b:undo_ftplugin = ""
```

Comment:
@brammool might have handled this but `undo_ftplugin` should be set to 'undo' the change to `commentstring`.

```vim
let b:undo_ftplugin = "setlocal commentstring<"
```

Reply:
Yes, I changed it in my copy.  Please also update the original.  I also made some tweaks to the syntax file.

Reply:
Sorry, do you mean that this PR still needs to be updated @brammool?

Reply:
You appear to quote a header where there is an "Original Author" entry in the header, but what I have only has the "Maintainer" entry with Mohamed Boughab.  Please check what the correct header is.

Reply:
I see that the headers are not the same everywhere, I'll open a new PR to fix this. The correct one is in `runtime/ftplugin/i3config.vim`

Reply:
https://github.com/vim/vim/pull/9467